tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/OpenTelemetrySdkTests.cs

```diff
@@ -204,6 +204,13 @@ private static string GetSuffix(string packageVersion, bool legacyOperationNames
                 return "_up_to_1_5_0" + legacyEnabled;
             }
 
+            // v1.7.0 fixed StartRootSpan to not be a child of the active span
+            if (!string.IsNullOrEmpty(packageVersion)
+             && new Version(packageVersion) <= new Version("1.7.0"))
+            {
+                return "_up_to_1_7_0" + legacyEnabled;
+            }
```

Comment:
Just wondering about the naming here - presumably 1.8.0 will have the same snapshots as 1.7.0, right? In that case, will the snapshots break 🤔

Reply:
I think this should only be the package range of `1.5.0` and `1.6.0` so [1.5.0, 1.7.0)
If `1.8.0` comes out it won't change the `_up_to_1_7_0` snapshots but will for the snapshots without a version, so those would break, but these should remain the same (I think) 

But yeah I'm not a huge fan of the naming/versioning scheme for these snapshots though

Reply:
I now see that I had `<=` instead of `<` so should be fine now